cannon/mipsevm/multithreaded/testutil/expectations_test.go

```diff
@@ -121,6 +127,12 @@ func TestValidate_shouldCatchMutations(t *testing.T) {
 	}
 }
 
+func findInactiveThread(e *ExpectedState) *ExpectedThreadState {
+	threads := e.threadExpectations.allThreads()
+	idx := int(len(threads) / 2)
```

Comment:
if there's a single thread left, this will incorrectly return that as inactive. It would be simpler to return the top of the inactive thread stack.